synapse/tests/files/stormpkg/testpkg.yaml

```diff
@@ -22,6 +22,25 @@ docs:
     - title: Foo Bar
       path: docs/foobar.md
 
+perms:
+    - perm: [ power-ups, testpkg, user ]
+      gate: cortex
+      desc: Controls user access to testpkg.
+      scopes: [ global, self ]
```

Comment:
What does it mean for perms declarations to have scope?

Reply:
Scope should be on vars not perms